s3_to_redshift.go

```diff
@@ -26,6 +27,7 @@ var (
 	dataDate        = flag.String("date", "", "data date we should process, must be full RFC3339")
 	configFile      = flag.String("config", "", "schema & table config to use in YAML format")
 	gzip            = flag.Bool("gzip", true, "whether target files are gzipped, defaults to true")
+	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
```

Comment:
For record-keeping: Christian and I chatted about this in person, and agreed that for now we ought to keep the default behavior (since changing it would perform a backwards incompatible change). However, he did bring up the point that this would be a potential stumbling block for users (who might not expect to know how to use `granularity`). As a result, documentation has been added in dd0fdc4